tests/pytest_framework/src/syslog_ng_config/renderer.py

```diff
@@ -41,6 +41,8 @@ def __render(self, re_create_config=None):
             self.__render_version()
         if self.__syslog_ng_config["sources"]:
             self.__render_statements(root_statement="sources", statement_name="source")
+        if self.__syslog_ng_config["filters"]:
```

Comment:
For now this looks good, but I am afraid we will run into trouble when rewrite rules are introduced, because they freely in mix into filters (ordering is undefined).

Reply:
Correct, we should think about this later.